secator/celery.py

```
@@ -403,6 +403,20 @@ def break_task(task, task_opts, results=[]):
 	# Clone opts
 	base_opts = task_opts.copy()
 
+	# Adjust rate_limit for chunked tasks
+	# Divide by chunk count but ensure it's at least 1
+	# Skip if rate_limit is 0 or None (no rate limiting)
+	if 'rate_limit' in base_opts and base_opts['rate_limit']:
```

Ruff (0.15.9)

[warning] Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)